qemu/tests/cfg/cpu_device_hotplug.cfg

```diff
@@ -0,0 +1,88 @@
+- cpu_device_hotplug_test: install setup image_copy unattended_install.cdrom
+    mem = 4096
+    virt_test_type = qemu
+    type = cpu_device_hotplug
+    cpu_hotplug_timeout = 2000
+    n_cpus_add = 1
+    kill_vm = yes
+    iterations = 5
+    onoff_iterations = 100
+    login_timeout = 240
+    kill_vm = yes
+    smp = 8
+    vcpu_maxcpus = 240
+    auto_cpu_model = "no"
+    cpu_hotplug_cmd = 'device_add CPU_MODEL-spapr-cpu-core,id=core{0},core-id={0}'
+    cpu_hotunplug_cmd = 'device_del core{0}'
+    variants:
+        - add_one_cpu:
+
+        - add_32_cpu:
+            n_cpus_add = 32
+        - add_64_cpu:
+            n_cpus_add = 64
+        - add_96_cpu:
+            n_cpus_add = 96
+        - add_128_cpu:
+            n_cpus_add = 128
+        - add_160_cpu:
+            n_cpus_add = 160
+        - add_192_cpu:
+            n_cpus_add = 192
+        - add_224_cpu:
+            n_cpus_add = 224
+        - add_max_cpu:
+            n_cpus_add = 240
+            pserial:
```

Comment:
@luckyh Making this test to work with only pseries ... adding
only ppc64le
in the config file.